### ISSUE TITLE
Docker Deployment

### ISSUE THREAD

**Issue by frostieDE** [SchulIT/icc#388]
Es wäre praktisch, wenn man die Anwendung auch als Docker Container bereitstellen könnte.

**Comment by SimonFrank14** [comment]
Liegen die Styles für die CSS Dateien in einem extra Verzeichnis? Diese könnte man dann auch mounten, da beim Mounting eines externen Volumes die internen Dateien des Containers immer mit denen vom gemounteten Volume an der Stelle überschrieben werden.

**Comment by SimonFrank14** [comment]
Kannst du mir ein paar Infos zu dem CMD geben?

`php bin/console bazinga:js-translation:dump assets/js/ --merge-domains`

Ich bekomme hier im Docker Image logischerweise einen Fehler, weil der scheinbar während des Builds eine DB Verbindung braucht und die nicht da ist. Stimmt das?

**Comment by frostieDE** [comment]
Oups.. Hier wird vermutlich das Problem sein, dass er eine Datenbankverbindung für die Logs braucht (obwohl ich eigentlich dachte, dass die Logs in einer Datei landen...). Für das eigentliche Command braucht es natürlich keine Verbindung zur Datenbank..

Hier muss ich mir mal überlegen, wie man das umgehen könnte... :)

**Comment by frostieDE** [comment]
Eventuell müsste die Lösung so aussehen, dass die Assets nicht vorab kompiliert werden sondern quasi beim Starten des Containers. Dann müsste aber natürlich auch Node und NPM im Image installiert sein.. Schick ist das aber natürlich nicht. 

Aber am Ende liegt das auch an dem doch eher bescheidenen Assetmanagement im ICC. Eigentlich möchte ich Webpack irgendwann loswerden (Symfony empfiehlt die Verwendung des AssetMappers), aber dafür fehlte bisher die Zeit... Damit würde sich das Problem vermutlich erübrigen :)

**Comment by SimonFrank14** [comment]
Ja, das funktioniert. So habe ich es aktuell auch lokal bei mir aktuell gelöst. Aber ja schön ist das wirklich nicht.
